test/foundry/OwnableTwoSteps.t.sol

```diff
@@ -40,14 +39,71 @@ contract OwnableTwoStepsTest is TestParameters, TestHelpers, OwnableTwoStepsErro
         emit InitiateOwnershipTransfer(_OWNER, newOwner);
         ownableTwoSteps.initiateOwnershipTransfer(newOwner);
         assertEq(ownableTwoSteps.potentialOwner(), newOwner);
+        assertEq(uint8(ownableTwoSteps.status()), uint8(Status.TransferInProgress));
```

Comment:
It should be the same without casting both values into uint8?

Reply:
the `assertEq` doesn't support custom Enum, it seems

Reply:
oh yeah you are right